src/test/codegen/stack-probes.rs

```diff
@@ -1,22 +0,0 @@
-// ignore-arm
-// ignore-aarch64
-// ignore-mips
-// ignore-mips64
-// ignore-powerpc
-// ignore-powerpc64
-// ignore-powerpc64le
-// ignore-riscv64
-// ignore-s390x
-// ignore-sparc
-// ignore-sparc64
-// ignore-wasm
-// ignore-emscripten
-// ignore-windows
-// compile-flags: -C no-prepopulate-passes
-
-#![crate_type = "lib"]
-
-#[no_mangle]
-pub fn foo() {
-// CHECK: @foo() unnamed_addr #0
```

Comment:
welp

Reply:
A little history on that -- commit cd2580722375671fa2967661f65b7b33570547ec changed this test from checking `"__rust_probestack"` to `"inline-asm"`, but then db4c54443499d35c5273149ab8694247bd6ba727 fully dropped that check instead of changing it back.